sanic/response.py

```diff
@@ -317,13 +333,37 @@ async def file(
     :param mime_type: Specific mime_type.
     :param headers: Custom Headers.
     :param filename: Override filename.
+    :param last_modified: The last moditied date and time of the file.
+    :param max_age: Max age for cache control.
     :param _range:
     """
     headers = headers or {}
     if filename:
         headers.setdefault(
             "Content-Disposition", f'attachment; filename="{filename}"'
         )
+    if isinstance(last_modified, datetime):
+        last_modified = last_modified.timestamp()
+
+    if last_modified is None:
+        last_modified = Path(location).stat().st_mtime
+    if isinstance(max_age, Default):
+        max_age = 0  # Should change this (default) value to configable?
```

Comment:
Yes.

Reply:
Working on this.

Reply:
I think `conifg` is not accessible in `file` function because nether `app` nor `app.config` is being passed into the function. After exploring how werkzeug did it, I came up with a similar behavior that returning `no-cache` (the response can be stored in caches, but the response must be validated with the origin server before each reuse) when `max-age` is not set with an extra option for `no-store` (absolutely not to cache). Can you re-review these changes? Much appreciate.